Applications/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ if (MODULE_Registration)
   endmacro()
 
   add_registration_command(register LibIO OPTIONAL LibLBFGS LibPointSet LibDeformable)
+  add_registration_command(create-sdm LibIO)
```

Comment:
I can centainly put it somewhere else, but as I mention above, it has nothing to do with FFD or DOFs